### PR TITLE
Dump: update from '-device vmcoreinfo' to 'vmcoreinfo = yes' 

### PR DESCRIPTION
**Pull request by Kemama**
ID:2155404

Signed-off-by: Ke MA <mama@redhat.com>

### CODE REVIEW

**Comment by Kemama** [comment]
Hi @nickzhq @leidwang 
Pls help me review this patch. 
Thanks!

File: qemu/tests/cfg/dump_guest_core.cfg [hunk 1]
```diff
@@ -14,7 +14,7 @@
     crash_cmd = "crash -i ${crash_script} /usr/lib/debug/lib/modules/%s/vmlinux ${vmcore_file}"
     dump_guest_memory_file = "/usr/share/qemu-kvm/dump-guest-memory.py"
     x86_64:
-        extra_params += " -device vmcoreinfo"
+        extra_params += " -device '{"driver":"vmcoreinfo"}'"
```
Comment (leidwang) [inline qemu/tests/cfg/dump_guest_core.cfg:17]: Recommend to use `vmcoreinfo = yes` to replace `extra_params += " -device '{"driver":"vmcoreinfo"}'" `

**Review by leidwang — DISMISSED** [review]

**Comment by vivianQizhu** [comment]
@Kemama Please update your commit title with `dump: `, and now it it not extra_params, the content need to be updated.

**Comment by Kemama** [comment]
> @Kemama Please update your commit title with `dump: `, and now it it not extra_params, the content need to be updated.

Yes. Done.

**Review by vivianQizhu — APPROVED** [review]

**Pull request merged**